### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by dfens**

### CODE REVIEW

**Comment by samuil** [comment]
I don't think it is desirable change.

**Comment by koleksiuk** [comment]
Please merge

**Comment by jmatraszek** [comment]
nglb3

File: index.html [hunk 1]
```diff
@@ -10,6 +10,23 @@
...
     <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <!--
+ ⊂ヽ
+　 ＼＼
+　　 ＼( ° ͜ʖ ͡° )
+　　　 >　⌒ヽ
+　　　/ 　 へ＼
+　　 /　　/　＼＼
+c==3 ﾚ　ノ　　 ヽつ
```
Comment (glock45) [inline index.html]: 👍

Comment (glock45) [inline index.html]: +

**Pull request closed without merging**